custom_components/waste_collection_schedule/waste_collection_schedule/source/muenchenstein_ch.py

```diff
@@ -0,0 +1,77 @@
+import json
+from datetime import datetime, timedelta
+
+import requests
+from bs4 import BeautifulSoup
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+TITLE = "Abfallsammlung Münchenstein"
```

Comment:
Remove Abfallsammlung - all swiss sources just use the name of the municipality.